pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -130,17 +161,93 @@ func (o *GinkgoRunSuiteOptions) Run(suite *TestSuite, junitSuiteName string, mon
 		return fmt.Errorf("failed reading origin test suites: %w", err)
 	}
 
-	fmt.Fprintf(o.Out, "found %d tests for suite\n", len(tests))
+	fmt.Fprintf(o.Out, "Found %d tests for in openshift-tests binary for suite %q\n", len(tests), suite.Name)
 
 	var fallbackSyntheticTestResult []*junitapi.JUnitTestCase
 	if len(os.Getenv("OPENSHIFT_SKIP_EXTERNAL_TESTS")) == 0 {
-		buf := &bytes.Buffer{}
-		fmt.Fprintf(buf, "Attempting to pull tests from external binary...\n")
-		externalTests, err := externalTestsForSuite(ctx)
+		// A registry of available external binaries and in which image
+		// they reside in the payload.
+		externalBinaries := []externalBinaryStruct{
+			{
+				imageTag:   "hyperkube",
+				binaryPath: "/usr/bin/k8s-tests",
+			},
+		}
+
+		var (
+			externalTests []*testCase
+			wg            sync.WaitGroup
+			resultCh      = make(chan externalBinaryResult, len(externalBinaries))
+			err           error
+		)
+
+		oc := util.NewCLIWithoutNamespace("default")
+		jobType, err := platformidentification.GetJobType(context.Background(), oc.AdminConfig())
```

Comment:
This will fail in MicroShift (no support for `platformidentification` because we are missing some of the apigroups), can you add a condition to skip it if its not supported?

Reply:
From https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/29025/pull-ci-openshift-origin-master-e2e-aws-ovn-microshift/1842228691740921856